packages/vitest/src/node/core.ts

```diff
@@ -145,6 +150,8 @@ export class Vitest {
       const serverRestart = server.restart
       server.restart = async (...args) => {
         await Promise.all(this._onRestartListeners.map(fn => fn()))
+        this.report('onServerRestart', 'config')
```

Comment:
Is the `'config'` reason here correct? On the line above it's not passed to `_onRestartListeners` calls like we do below on `server.watcher.on('change', ...` callback. 

```suggestion
        this.report('onServerRestart')
```

This affects here:

https://github.com/vitest-dev/vitest/blob/0d67f04b653f7a5f6971083d05367e5a2478a17d/packages/vitest/src/node/reporters/base.ts#L364-L374

Reply:
Yeah, I just didn't notice